packages/destination-actions/src/destinations/insider/cartViewedEvent/index.ts

```diff
@@ -11,13 +11,21 @@ import {
   user_attributes,
   uuid
 } from '../insider-properties'
-import { API_BASE, sendTrackEvent, UPSERT_ENDPOINT } from '../insider-helpers'
+import { API_BASE, sendBulkTrackEvents, sendTrackEvent, UPSERT_ENDPOINT } from '../insider-helpers'
 
 const action: ActionDefinition<Settings, Payload> = {
   title: 'Cart Viewed Event',
   description: 'Record Cart Viewed Event to Insider',
   defaultSubscription: 'type = "track" and event = "Cart Viewed"',
   fields: {
+    enable_batching: {
+      type: 'boolean',
+      label: 'Send Batch Request',
+      description:
+        'When enabled, the action will send a batch request to Insider. Batches can contain up to 1000 records in a request.',
+      required: true,
+      default: false
```

Comment:
can it be default true ? for partners

Reply:
can be, segment says it can be a delay a bit for batch, because of that I think partner can decide it